src/styles/main.scss

```diff
@@ -1,3 +1,200 @@
-body {
+body,
+p,
+a,
+h2 {
+  padding: 0;
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  color: #060b35;
```

Comment:
In my opinion, you can use to to reduce repetition, for example, fonts color, cards width etc...